nativelib/src/main/resources/dirent.c

```diff
@@ -37,3 +37,14 @@ int scalanative_readdir(DIR *dirp, struct scalanative_dirent *buf) {
 }
 
 int scalanative_closedir(DIR *dirp) { return closedir(dirp); }
+
+int scalanative_gettempdir(char *buffer, size_t length)
+{
+#ifndef _WIN32
+  const char* tmpdir = "/tmp";
+  strncpy(buffer, tmpdir, length);
+  return 0;
+#else
+  return getWinTempDir(buffer, length);
```

Comment:
I don't mind to use it, where this system property was initialized btw? I can't find the place in code.
btw, on win10 temp folder inside windows folder is restricted, so it should point somewhere to "C:\Users\muxanick\AppData\Local\Temp\".